engines/instance_verification/lib/instance_verification/engine.rb

```diff
@@ -72,6 +72,52 @@ def self.set_cache_inactive(cache_key, mode)
     InstanceVerification.update_cache(cache_key, mode)
   end
 
+  def self.verify_instance(request, logger, system)
+    return false unless request.headers['X-Instance-Data']
+
+    base_product = system.products.find_by(product_type: 'base')
+    return false unless base_product
+
+    # check the cache for the system (20 min)
```

Comment:
s/(20 min)// the time of cache expiration is controlled elsewhere as such adding the time here as a hard coded value is a potential source of conveying inaccurate information in a comment. Remove the value or refer to where the cache time is set.